types/markdown-it-emoji/index.d.ts

```diff
@@ -1,18 +1,19 @@
-// Type definitions for markdown-it-emoji 1.4
+// Type definitions for markdown-it-emoji 2.0
 // Project: https://github.com/markdown-it/markdown-it-emoji
 // Definitions by: Christopher Quadflieg <https://github.com/Shinigami92>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import { PluginSimple, PluginWithOptions } from 'markdown-it/lib';
 
-import { PluginSimple, PluginWithOptions } from 'markdown-it';
+export as namespace markdownitEmoji;
 
-declare namespace MarkdownItEmoji {
+declare namespace markdownitEmoji {
```

Comment:
Why do we want lowercase started namespaces? Isn't it the default to use `PascalCase` for namespaces?
Looking into official resources uses these style: https://www.typescriptlang.org/docs/handbook/namespaces.html

Reply:
I understand, but:
1) i'd like to follow original naming:
https://github.com/markdown-it/markdown-it-emoji/blob/master/dist/markdown-it-emoji.js#L2048
2) namespace if for a function, not for a internal module like, so more like CJS namespacing from DT template:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html
(described function starts lowercase)

Reply:
So these reasons are okay-ish for me